webnn/idlharness.https.any.js

```diff
@@ -0,0 +1,40 @@
+// META: global=window,worker
```

Comment:
```suggestion
// META: global=window,dedicatedworker
```
WebNN only support `DedicatedWorker`.